src/core/bittorrent/sessionstatus.h

```diff
@@ -51,6 +51,16 @@ namespace BitTorrent
         // account "useful" part of the rate
         int payloadUploadRate() const;
 
+        // Additional download/upload rates
+        int uploadRate() const;
+        int downloadRate() const;
+        int ipOverheadUploadRate() const;
+        int ipOverheadDownloadRate() const;
+        int dhtUploadRate() const;
+        int dhtDownloadRate() const;
+        int trackerUploadRate() const;
+        int trackerDownloadRate() const;
```

Comment:
@glassez this hooks into classes you're more familiar with. Do you have any objections with this commit?